### DISCUSSION TITLE
How to create a station?

### DISCUSSION THREAD

**Discussion by Simomagy** [ChAoSUnItY/EkiLib#11] [Q&A] [answered]
Hello, i just need to know how to create a station. I don't know how the station tuner works (it says "Invalid item! It may not contains required values") and when i opne a gate with my ticked mod says "Exit from unknown station, entered from unknown station". Also i read the previous Q&A where a guy asked how to edit the station sign but i don't know how to "open the station tab". 
Thank you in advance for help. 

P.S. mod is awesome keep working please :)

**Comment by ChAoSUnItY** [comment, marked as answer]
Sorry for late reply. To create a station, press L key in default will open a station list screen, there's a button shows "Create / Modify", if you press it without select any station above, you'll get a screen for station creation, the position of station is based on your position, the rest can be modified without limitation; If you press it with a station selected above, you'll get a screen for station modification instead, not every field for station property is mutable.

For station tuner, heads to the selection screen, and select a station first, you'll see the button at right bottom will be enable after you selected, but ensure you're holding a station tuner, otherwise it'll still unavailable, you can drag your mouse on the button to see what's wrong if it is unavailable, it'll show the detail of what you're doing wrong.

After you bind the selected station to your tuner, you can right click it with Station Nameplate, Ticket Gate, or Ticket Vendor, it'll memorize the station your tuner is bound to.

**Reply by Simomagy** [reply]
Thnak you man, great mod keep it up please
